### DISCUSSION TITLE
cant sync after upgrade

### DISCUSSION THREAD

**Discussion by geordiepaul101** [Chia-Network/chia-blockchain#11805] [Support]
Hi,
I was on 1.2.11 which worked fine until it stopped syncing recently, so I upgraded to 1.3.5 and this too cannot sync. It says "syncing 0/2000000" (approx) but never actually changes from this. I have waited days, with no joy. I have looked all over and cannot fathom waht it might be, please help!

Thanks

**Comment by loppefaaret** [comment]
it might be a blockchain corrupted problem, without knowing more... 
if you could attach your `debug.log` found in you users home folder, in `~\.chia\mainnet\log\` i might be able to verify what is going on
